core/templates/dev/head/player/PlayerServices.js

```diff
@@ -692,3 +748,70 @@ oppia.directive('feedbackPopup', ['oppiaPlayerService', function(oppiaPlayerServ
     }]
   };
 }]);
+
+oppia.controller('InformationCard', function ($scope, $modal) {
+
+  $scope.showInformationCard = function () {
+
+    var modalInstance = $modal.open({
+      animation: true,
+      templateUrl: 'popover/informationCard',
+      windowClass: 'oppia-modal-information-card',
+      controller: ['$scope', '$http', '$modal', '$modalInstance','oppiaPlayerService', 'ratingService',
+                   'oppiaHtmlEscaper', 'embedExplorationButtonService',
+                    function (
+                      $scope, $http, $modal, $modalInstance, oppiaPlayerService, ratingService, oppiaHtmlEscaper,
+                      embedExplorationButtonService) {
+
+        var _informationCardData = oppiaPlayerService.getInformationCardData();
+        $scope.serverName = window.location.protocol + '//' + window.location.host;
+        $scope.escapedTwitterText = oppiaHtmlEscaper.unescapedStrToEscapedStr(
+          GLOBALS.SHARING_OPTIONS_TWITTER_TEXT);
+        $scope.showEmbedExplorationModal = embedExplorationButtonService.showModal;
+
+        $scope.explorationId = oppiaPlayerService.getExplorationId();
+        $scope.infoCardBackgroundImage = {
+          'background-image': 'url('+ _informationCardData.infoCardImageUrl +')'
+        };
+        $scope.explorationRatings = ratingService.getUserRating() !== null ? ratingService.getUserRating() : 0;
+        $scope.explorationTags = _informationCardData.explorationTags;
+        $scope.explorationDescription = _informationCardData.explorationObjective;
```

Comment:
Can we capitalize the first letter? We already do this in the gallery; it might be a good idea to extract the logic into a common filter and put it in filters.js.